ray-on-gke/cloudbuild.yaml

```diff
@@ -1,17 +1,71 @@
 steps:
   - id: 'tf validate platform'
-    name: 'hashicorp/terraform:1.0.0'
+    name: 'gcr.io/$PROJECT_ID/terraform'
     script: |
-      cd platform
       terraform init
-      terraform validate 
+      terraform validate
+    dir: 'ray-on-gke/platform/'
     waitFor: ['-']
-  
+
+  - id: 'tf apply platform'
+    name: 'gcr.io/$PROJECT_ID/terraform'
+    entrypoint: 'sh'
+    args:
+      - '-c'
+      - |
+        terraform apply -var=project_id=$PROJECT_ID \
+        -var=cluster_name=cluster-$SHORT_SHA-$_PR_NUMBER \
```

Comment:
$_PR_NUMBER will be empty if we don't trigger from PR. But a empty string will break cluster naming rules. 
I change the cluster name to ray-$SHORT_SHA-$_PR_NUMBER-cluster. 

Trying to default $_PR_NUMBER to other values but it will actually get changed when trigger from PR.